Dockerfile

```diff
@@ -81,48 +103,36 @@ RUN cd clouseau_deps \
 # now we can add all source code and start compiling
 RUN cd /clouseau \
   && git clone -b ntr_master https://github.com/neutrinity/clouseau . \
-  && cp -RT /clouseau_deps/ /clouseau/ && rm -r /clouseau_deps
+  && cp -RT /clouseau_deps/ "${CLOUSEAU_PATH}/" && rm -r /clouseau_deps
 
-RUN chown -R couchdb:couchdb /clouseau /couchdb
-USER couchdb
+RUN chown -R couchdb:couchdb $CLOUSEAU_PATH /couchdb
 
 # TODO tests need to get unskipped
 RUN  cd /clouseau && mvn verify -Dmaven.test.skip=true
 
-USER root
-
-# Cleanup build detritus
-RUN apt-get purge -y --auto-remove apt-transport-https \
-  gcc \
-  g++ \
-  libcurl4-openssl-dev \
-  libicu-dev \
-  libmozjs185-dev \
-  make \
-  && rm -rf /var/lib/apt/lists/* /usr/src/couchdb*
+# FIXME: this is for clouseau's start-script
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+  && apt-get -qq install -y nodejs
 
-COPY ./config/local.ini /couchdb/etc/local.d/
-COPY ./config/vm.args /couchdb/etc/
-RUN chown -R couchdb:couchdb /couchdb/etc/local.d/ /couchdb/etc/vm.args
+COPY ./config/local.ini "$COUCHDB_PATH/etc/local.d/"
+COPY ./config/vm.args "$COUCHDB_PATH/etc/"
+RUN chown -R couchdb:couchdb "$COUCHDB_PATH/etc/local.d/" "$COUCHDB_PATH/etc/vm.args"
 
-RUN mkdir /couchdb/data
-VOLUME ["/couchdb/data"]
+RUN mkdir "$COUCHDB_PATH/data"
+VOLUME ["$COUCHDB_PATH/data"]
 
 EXPOSE 5984
 
-WORKDIR /couchdb
+WORKDIR $COUCHDB_PATH
 
-COPY ./start-couchdb /couchdb/
-RUN chmod +x /couchdb/start-couchdb
-COPY ./start-clouseau /couchdb/
-RUN chmod +x /couchdb/start-clouseau
+COPY ./start-couchdb $COUCHDB_PATH
+COPY ./start-clouseau $COUCHDB_PATH
 
 # Setup directories and permissions
-RUN chown -R couchdb:couchdb /couchdb
-
-USER couchdb
+RUN chmod +x start-couchdb && chmod +x start-clouseau && chown -R couchdb:couchdb $COUCHDB_PATH
 
-RUN mkdir -p /clouseau/target/clouseau1
-VOLUME ["/clouseau/target/clouseau1"]
+RUN mkdir -p "$CLOUSEAU_PATH/target/clouseau1"
+VOLUME ["$CLOUSEAU_PATH/target/clouseau1"]
 
+USER couchdb
 ENTRYPOINT ["/couchdb/start-couchdb"]
```

Comment:
i currently get permission errors when trying to run this image in the helm chart for the 
chown command "chown -R couchdb:couchdb /couchdb" in /couchdb/start-couchdb
this is not broken by this PR as the error is also thrown in the base branch but it would help if we could also fix this first to test?